src/dcd/server/autocomplete/util.d

```diff
@@ -738,3 +738,16 @@ unittest
 	i = skipParenReverseBefore(t, i, tok!")", tok!"(");
 	assert(i == 1);
 }
+
+AutocompleteResponse.Completion makeSymbolCompletionInfo(const DSymbol* symbol, char kind)
+{
+	string definition;
+	if ((kind == 'v' || kind == 'm') && symbol.type)
+		definition = symbol.type.name ~ ' ' ~ symbol.name;
+	else if (kind == 'e')
```

Comment:
Any reason for using 'v', 'm', and 'e' here instead of their names from the enum?